pkg/infra/pulumi_aws/iac/api_gateway.ts

```diff
@@ -160,6 +160,10 @@ export class ApiGateway {
                 stageVariables: {
                     vpcLinkId: vpcLink.id,
                 },
+                // Vary the description to force recreation if the vpcLink changes
+                // https://github.com/hashicorp/terraform/issues/6613#issuecomment-322264393
+                // https://github.com/hashicorp/terraform/issues/18392#issuecomment-402684841
+                description: pulumi.interpolate`Stage hash: ${sha256.sync(vpcLink.id)}`,
```

Comment:
this is for v2API, the restAPI we use is v1, which is just in the create Docker Lambda function